go/vt/orchestrator/db/generate_patches.go

```diff
@@ -513,7 +513,7 @@ var generateSQLPatches = []string{
 	`
 		ALTER TABLE
 			database_instance
-			ADD COLUMN master_uuid varchar(64) CHARACTER SET ascii NOT NULL AFTER oracle_gtid
+			ADD COLUMN primary_uuid varchar(64) CHARACTER SET ascii NOT NULL AFTER oracle_gtid
```

Comment:
Should be `source_uuid`: it's the `@@server_uuid` of the immediate _source_, not the topology's `primary`.